packages/orbit-components/src/Textarea/Textarea.stories.tsx

```diff
@@ -7,185 +8,114 @@ import { SPACINGS_AFTER } from "../common/consts";
 
 import Textarea from ".";
 
-export default {
+const meta: Meta<typeof Textarea> = {
   title: "Textarea",
-};
-
-export const Default = ({ label, value, placeholder }) => {
-  return (
-    <Textarea label={label} placeholder={placeholder} onChange={action("change")} value={value} />
-  );
-};
+  component: Textarea,
 
-Default.story = {
   parameters: {
-    info: "Some description about this type of textarea in general.",
+    info: "Textarea component. Check Orbit.Kiwi for more detailed guidelines.",
+    controls: {
+      exclude: ["onChange"],
+    },
   },
-};
-
-Default.args = {
-  label: "Label",
-  value: "",
-  placeholder: "Placeholder",
-};
-
-export const WithLabel = ({ label, value, placeholder }) => {
-  return (
-    <Textarea label={label} placeholder={placeholder} onChange={action("change")} value={value} />
-  );
-};
-
-WithLabel.story = {
-  name: "With label",
 
-  parameters: {
-    info: "Some description about this type of textarea in general.",
+  args: {
+    name: "Textarea",
+    value: "Textarea",
+    onChange: action("onChange"),
   },
 };
 
-WithLabel.args = {
-  label: "Label",
-  value: "",
-  placeholder: "Placeholder",
-};
+type Story = StoryObj<typeof Textarea>;
 
-export const WithHelp = ({ value, placeholder, help }) => {
-  return (
-    <Textarea placeholder={placeholder} help={help} onChange={action("change")} value={value} />
-  );
-};
+export default meta;
 
-WithHelp.story = {
-  name: "With help",
+export const Default: Story = {};
```

Comment:
"my" settings for the default story are based on MDN/W3scools websites as there is a `name` specified in the default example (also id, row, cols - but we have a standard to put `id` only into playground stories, `rows` and `cols` are not used in our component). on the other side, it's true that the `value` is not in my default story there.

---

Overall, from my POV, it makes sense to add a `placeholder` as you suggest, on the other side I wouldn't put there `label` as it's "separated" component (`span`), not necessary for rendering the textarea itself. WDYT?

Reply:
On one hand, I understand you reasoning, but on the other hand, e.g. in `InputFIeld` we put just `label`, `value` and `placeholder` into the default story 🤷 

> I wouldn't put there label as it's "separated" component

It might be separated element from input textarea, but it's part of `Textarea` component 🤔 Looking at it now and at other stories, it would maybe make the most sense to me to include `name`, `label`, `value` and `placeholder` here as default and just remove the `WithLabel` story. WDYT?

Reply:
yeah, might be. I don't have any strong preference, just described my thoughts.